ga4gh/cli.py

```diff
@@ -270,7 +380,17 @@ def client_main():
         help="Benchmark server performance")
     bmParser.set_defaults(runner=BenchmarkRunner)
     addUrlArgument(bmParser)
-    addOptions(bmParser)
+    addVariantSearchOptions(bmParser)
+
+    # variants/search
+    vsParser = subparsers.add_parser(
```

Comment:
Yeah, I was wondering what the best thing would be to do here.  Presumably we could at some point want a handle on each of the parsers at some future point... for a refactor or testing or something.  Furthermore, I'm worried about calling a method on the wrong parser, seeing as how this is a really long method, and at least different names gives some pseudo-scoping, albeit not much.

One solution I was thinking about was wrapping each subparser creation and configuration in an inner method in `client_main` and then invoking it.  But that just seemed like too much work for not much gain.  Testing mitigates some of the risk.

I think it's best if we just leave it as it is, unless someone proposes a clearly superior solution.